site/kube-addon.md

```diff
@@ -55,6 +55,32 @@ Shut down Kubernetes, and _on all nodes_ perform the following:
 Then relaunch Kubernetes and install the addon as described
 above.
 
+**Note:** You can customise the generated YAML file by passing Weave Net's options, arguments and environment variables as query parameters:
+
+  - `version`: Weave Net's version. Default: latest release.
```

Comment:
I've just noticed this could break for previous versions of Net, given the YAML has evolved a bit.
1. Do we want to maintain backward compatible logic for versions of Net too?
2. I doubt we do so maybe we should just not document this query parameter?